public/styles/components/_accordion.scss

```diff
@@ -110,6 +110,21 @@
         content: '';
       }
 
+      // TODO: (1) Remove this code when icon library is live (second part below).
+      &::after {
+        position: absolute;
+        left: 0;
+        align-self: stretch;
+        width: var(--space-6x);
+        margin-top: 5px;
```

Comment:
if we can't use a space token here, we should use a one-off rem value. 

i also believe our linters should fail the build on hardcoded px values. can we confirm? if it's not failing, we'll need to look into settings.